src/vlinder/presentation.py

```diff
@@ -0,0 +1,39 @@
+from fpdf import FPDF
+
+
+class PDF(FPDF):
+    def chapter_title(self, title, rgb):
+        self.set_font("Arial", "B", 16)
+        self.set_text_color(rgb[0], rgb[1], rgb[2])
+        self.multi_cell(0, 10, title, 0, "C")
+        self.ln(10)
+
+    def chapter_subtitle(self, subtitle):
+        subtitle = subtitle.replace("‘", "'")
+        subtitle = subtitle.replace("’", "'")
+        self.set_font(family="Arial", size=12)
+        self.set_text_color(0, 0, 0)
+        self.multi_cell(0, 10, subtitle, 0, "L")
+        self.ln(10)
+
+    def titlepage_title(self, title, rgb):
```

Comment:
ik ga weer even mierenneuken, maar we hebben overal woord_woord(_woord), dus zou dat hier ook doenL title_page_title